azure-pipelines-linux-mac.yml

```diff
@@ -61,6 +61,10 @@ steps:
       yarn compile
     displayName: Compile
 
+  - script: |
+      yarn gulp install-sqltoolsservice
+    displayName: Install SQL Tools Service
```

Comment:
If we go with this you should also add

gulp install-ssmsmin

Reply:
yeah, I was thinking that adding only the tools service was a good "middle ground" between adding some test coverage in CI builds vs. adding extra steps to CI build.  Though I can see the argument to add ssms-min as well.  We'll discuss tomorrow.  All-up I think it's a bit of a redundant discussion since we're going to enable integration tests in CI builds, which will necessitate installing at least the tools service in any case.